packages/block-library/src/post-comments-form/index.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-comments-form` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-comments-form` block on the server.
+ *
+ * @return string Returns the filtered post comments form for the current post.
+ */
+function render_block_core_post_comments_form() {
+	$post = gutenberg_get_post_from_context();
+	if ( ! $post ) {
+		return '';
+	}
+	return comment_form( array(), $post->ID );
```

Comment:
Thinking we should be consistent in all of our blocks about supporting the custom className prop and having a block wrapper with the default className.

Reply:
As part of the common block wrapper that @ellatrix worked on?

Reply:
That could help making it consistent yes, but for now, we need to add it manually in "edit" functions (the className prop is available there though) and in the server, everything is manual

Reply:
:+1: